app/src/main/java/co/condorlabs/customcomponents/customimageview/AppCompactTouchImageView.kt

```diff
@@ -0,0 +1,372 @@
+package co.condorlabs.customcomponents.customimageview
+
+import android.animation.ValueAnimator
+import android.content.Context
+import android.graphics.PointF
+import android.graphics.drawable.Drawable
+import android.util.AttributeSet
+import android.view.MotionEvent
+import android.view.ScaleGestureDetector
+import androidx.appcompat.widget.AppCompatImageView
+import kotlin.math.abs
+import kotlin.math.max
+import kotlin.math.min
+
+/**
+ * Created by E Duque on 5/13/20.
+ * Condor Labs
+ * eduque@condorlabs.io
+ */
+class AppCompactTouchImageView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
```

Comment:
Remember that we talk about leaving a default `defStyleAttr`, why is this still here?
Refer to this:
https://proandroiddev.com/misconception-about-kotlin-jvmoverloads-for-android-view-creation-cb88f432e1fe

Reply:
because I don't need to keep imageview styles for this component...

Reply:
if that's the case then i suggest to remove that param of the constructor, according to the article 
```
Why does this “lazy” solution always work? Because we are using override on the 2-parameter constructor of the superclass, so whatever the default style that the superclass uses will still carry over
```

Reply:
Ok